chapters/002-smol-internet/README.md

```diff
@@ -107,50 +84,35 @@ What's a network without a container right? We start this chapter with a lone
       - NET_ADMIN
 ```
 
-**NOTE** We add `boudi` to `doggonet` by the end of the chapter. To follow the
-*course of this chapter, you may wish to remove the `doggonet` network from
-`boudi`.
+**NOTE** We add `boudi` to `doggonet` by the end of the chapter. To follow the *course of this chapter, you may wish to remove the `doggonet` network from `boudi`.
 
 ### Can our networks communicate with each other?
 
-Now we have 2 separate networks. Fantastic! An internet is a group of hosts on
-different networks that can all communicate with each other. We have the hosts,
-we have the networks, but before we go about getting them to talk to each other,
-let's make sure they can't already communicate... To do this, we're gonna reuse
-the same tricks we did in part 001.
+Now we have 2 separate networks. Fantastic! An internet is a group of hosts on different networks that can all communicate with each other. We have the hosts, we have the networks, but before we go about getting them to talk to each other, let's make sure they can't already communicate... To do this, we're gonna reuse the same tricks we did in part 001.
 
 First, let's jump onto one of the containers on our `squasheeba` network:
 
 ```
 hopon boudi
 ```
 
-We're going to try to ping this container from `tara` on the `doggonet` network.
-We can see if the ping reaches our container by running `tcpdump` and looking
-for any output.
+We're going to try to ping this container from `tara` on the `doggonet` network. We can see if the ping reaches our container by running `tcpdump` and looking for any output.
```

Comment:
```suggestion
We're going to try to ping `boudi` from `tara` on the `doggonet` network. We can see if the ping reaches our container by running `tcpdump` and looking for any output.
```

But also... can't we see if the `ping` reaches `boudi` by the presence (or lack thereof) of a response on `tara`?

Reply:
maybe we ignore that comment for now? it seems like this is a really simple intro to `tcpdump` that starts getting people comfortable. then we get to build on understanding that output in future chapters...